Formula/blucat.rb

```diff
@@ -1,12 +1,12 @@
 class Blucat < Formula
   desc "netcat for Bluetooth"
   homepage "http://blucat.sourceforge.net/blucat/"
-  url "http://blucat.sourceforge.net/blucat/wp-content/uploads/blucat-aa3e02.zip"
-  version "0.9"
-  sha256 "6dcd6bf538a06c2f29d21a9e94d859d91667a7014244462bffca9767bba5307d"
+  url "https://github.com/ieee8023/blucat/archive/v0.91.tar.gz"
+  sha256 "3e006d25b7e82689826c89ffbbfa818f8b78cced47e6d0647e901578d330a2f4"
+  head "https://github.com/ieee8023/blucat.git"
 
   depends_on "ant" => :build
-  depends_on :java => "1.6"
+  depends_on :java
```

Comment:
… it uses the newest available Java installed on the system. The version requirement is transparently forwarded to `/usr/libexec/java_home --version` and `java_home` implements the selection logic. If `--version` is omitted, there's neither a lower nor upper limit. (My gut feeling is that we should always specify a minimum as documented by upstream.)